pkgs/development/tools/rq/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, lib, fetchFromGitHub, rustPlatform, libiconv, llvmPackages, v8 }:
+{ stdenv, lib, fetchFromGitHub, rustPlatform, libiconv }:
```

Comment:
You can drop `libiconv` too, it's not needed for build anymore.

Note, you need to drop libiconv line for rq from `pkgs/top-level/all-packages.nix` too ...